### PR TITLE
Added scrollTo action.

### PR DESCRIPTION
**Pull request by sd-tomita**
## What's change ?
The page scrolling function was added to action.js.
Please merge to master branch, if you like.

## Sample
```js
actions: [
  {scrollTo: {x:100, y:150}}
]
```

### CODE REVIEW

File: src/actions.js [hunk 1]
```diff
@@ -151,3 +151,8 @@ export function authenticateAs(checker, action){
       .then(() => checker.driver.get(checker.lastUrl))
   })
 }
+
+export function scrollTo(checker, action){
+  const coord = action.scrollTo;
+  checker.driver.executeScript("window.scrollTo(0," + coord + ")");
```
Comment (gomo) [inline src/actions.js]: Your code only supports vertical scrolling, but please also support horizontal scrolling.

```js
{scrollTo: {x: 0, y: 100}}
```

I think that it is good to pass arguments as objects, and for axes omitted, set 0 to the default value.

**Review by gomo — CHANGES_REQUESTED** [review]

File: src/actions.js (hunk 1 above)
Comment (gomo) [reply src/actions.js]: Please write test too.

**Comment by sd-tomita** [comment]
@gomo 
Thank you for suggesting correction.
Fix for https://github.com/SunriseDigital/simple-selenium-checker/pull/3#pullrequestreview-63556158 .
Please check it out again.

**Review by gomo — APPROVED** [review]
LGTM 👍

**Pull request merged**